environment-dev.yml

```
@@ -6,6 +6,7 @@ dependencies:
   - codecov
   - ele
   - foyer>=0.9.4
+  - freud
```

I'd pin to `>=2.0.0` everywhere in this PR.
```suggestion
  - freud>=2.0.0
```